modules/javafx.graphics/src/main/java/javafx/application/Platform.java

```diff
@@ -432,4 +436,280 @@ public static ReadOnlyBooleanProperty accessibilityActiveProperty() {
         }
         return accessibilityActiveProperty.getReadOnlyProperty();
     }
+
+    /**
+     * Gets the preferences of the current platform.
+     * <p>
+     * The map returned from this method is unmodifiable, which means that keys and values cannot
+     * be added, removed, or updated. Calling any mutator method on the map will always cause
+     * {@code UnsupportedOperationException} to be thrown. However, the mappings will be updated
+     * by JavaFX when the operating system reports that a platform preference has changed.
+     *
+     * @return the {@code Preferences} instance
+     * @since 22
+     */
+    public static Preferences getPreferences() {
+        return PlatformImpl.getPlatformPreferences();
+    }
+
+    /**
+     * Contains UI preferences of the current platform.
+     * <p>
+     * {@code Preferences} extends {@link ObservableMap} to expose platform preferences as key-value pairs.
+     * For convenience, {@link #getInteger}, {@link #getDouble}, {@link #getBoolean}, {@link #getString},
+     * {@link #getColor}, {@link #getPaint}, and {@link #getValue} are provided as typed alternatives to
+     * the untyped {@link #get} method.
+     * <p>
+     * The preferences that are reported by the platform may be dependent on the operating system version,
+     * so applications should not assume that a particular preference is always available.
+     * <p>
+     * The following list contains all preferences that are potentially available on the specified platforms:
+     *
+     * <table>
```

Comment:
we might want to add an anchor somewhere here, to be able to create a link in Platform.getPreferences(), similar to what String.format() -> Formatter#syntax does:

```
<h3><a id="syntax">Format String Syntax</a></h3>
```

Reply:
I've added a `@see` tag to the `Platform.getPreferences()` method, which links to the table in the class documentation of `Platform.Preferences`.